src/test/java/com/hocs/test/pages/comp/COMPTriage.java

```diff
@@ -157,6 +165,10 @@ public void selectPermanentlyCloseCase(String yesNo) {
         clickTheButton("Confirm");
     }
 
+    public void openExGratiaAccordion() {
+        registration.openOrCloseAccordionSection("Ex-Gratia");
```

Comment:
as COMPTriage also extends BasePage, you shouldnt need the reference to the registration object to call the method here

Reply:
Changed